### PR TITLE
Ship OnlyEmojisInputFilter & MaximalNumberOfEmojisInputFilter.

### PR DESCRIPTION
**Pull request by vanniktech**

### CODE REVIEW

**Comment by mario** [comment]
Looks decent.

File: emoji/src/main/java/com/vanniktech/emoji/OnlyEmojisInputFilter.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+import android.text.InputFilter;
+import android.text.Spanned;
+import androidx.annotation.Nullable;
+
+public final class OnlyEmojisInputFilter implements InputFilter {
+  @Nullable private final Integer maxCount;
+
+  public OnlyEmojisInputFilter() {
+    maxCount = null;
```
Comment (rubengees) [inline emoji/src/main/java/com/vanniktech/emoji/OnlyEmojisInputFilter.java]: Isn't this a constraint which belongs into another Filter and maybe not even in the library?

Comment (vanniktech) [reply emoji/src/main/java/com/vanniktech/emoji/OnlyEmojisInputFilter.java]: Dunno. :man_shrugging: I feel like they can kinda live in the same InputFilter.

Comment (rubengees) [reply emoji/src/main/java/com/vanniktech/emoji/OnlyEmojisInputFilter.java]: I have no strong feeling about this, but these are two separate concerns.
Also, ìsn't it already possible to set the maxLength with `android:maxLength`?

Comment (vanniktech) [reply emoji/src/main/java/com/vanniktech/emoji/OnlyEmojisInputFilter.java]: You should know that Emojis have different lengths. This won't work. I'll try splitting them up - if that works it's even better.

Comment (vanniktech) [reply emoji/src/main/java/com/vanniktech/emoji/OnlyEmojisInputFilter.java]: I like it even more now. Thanks @rubengees 
Composition is a nice thing

Comment (rubengees) [reply emoji/src/main/java/com/vanniktech/emoji/OnlyEmojisInputFilter.java]: Ah you are right, completly forgot about emojis having multiple chars :man_facepalming:!

**Comment by vanniktech** [comment]
Waiting for your review @mario before merging

**Comment by vanniktech** [comment]
So what now @mario?

**Comment by mario** [comment]
Merged1

**Pull request merged**